Formula/k/kubefirst.rb

```diff
@@ -28,7 +28,7 @@ class Kubefirst < Formula
   depends_on "go" => :build
 
   def install
-    ldflags = "-s -w -X github.com/kubefirst/runtime/configs.K1Version=v#{version}"
+    ldflags = "-s -w -X github.com/kubefirst/kubefirst/configs.K1Version=v#{version}"
```

Comment:
The `kubefirst version` command is still using the `github.com/kubefirst/runtime/configs` package, rather than `github.com/kubefirst/kubefirst/configs`: https://github.com/kubefirst/kubefirst/blob/v2.3.5/cmd/version.go#L13

Reply:
Yeah I thought we already changed that in this pr. Thanks for calling it out. Seems to fix it locally.

Reply:
yeah this failed anyways.